tests/py/test_api_multi_gpu.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+import trtorch
+import torch
+import torchvision.models as models
+
+from multi_gpu_test_case import MultiGpuTestCase
+
+gpu_id = 1
+class TestCompile(MultiGpuTestCase):
+
+    def setUp(self):
+        self.input = torch.randn((1, 3, 224, 224)).to("cuda")
```

Comment:
Id say just put gpu_id here like `self.gpu_id = 1`

Reply:
I also need the trtorch.set_device(gpu_id) outside the class. Hence, re-used the global var. Single source of truth

Reply:
You should set device during setup. Also shouldnt this test that the device switches properly?